Extension/package.nls.json

```diff
@@ -185,8 +185,8 @@
     "c_cpp.configuration.preferredPathSeparator.markdownDescription": { "message": "The character used as a path separator for `#include` auto-completion results.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.simplifyStructuredComments.markdownDescription": { "message": "If `true`, tooltips of hover and auto-complete will only display certain labels of structured comments. Otherwise, all comments are displayed.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.commentContinuationPatterns.items.anyof.string.markdownDescription": { "message": "The pattern that begins a multiline or single line comment block. The continuation pattern defaults to ` * ` for multiline comment blocks or this string for single line comment blocks.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
-    "c_cpp.configuration.autocompleteDoxygenComment.description":  "Controls whether to automatically insert the Doxygen comment after typing the chosen comment style.",
-    "c_cpp.configuration.generatedDoxygenCommentStyle.description": "The string of characters used as the starting line of the Doxygen comment.",
+    "c_cpp.configuration.doxygen.generateOnType.description":  "Controls whether to automatically insert the Doxygen comment after typing the chosen comment style.",
```

Comment:
oh weird, these descriptions split the descriptions of commentContinuationPatterns